pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -179,6 +179,11 @@ func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, optio
 	output, err := p.ec2api.CreateLaunchTemplateWithContext(ctx, &ec2.CreateLaunchTemplateInput{
 		LaunchTemplateName: aws.String(launchTemplateName(options)),
 		LaunchTemplateData: &ec2.RequestLaunchTemplateData{
+			BlockDeviceMappings: []*ec2.LaunchTemplateBlockDeviceMappingRequest{{
+				Ebs: &ec2.LaunchTemplateEbsBlockDeviceRequest{
+					Encrypted: aws.Bool(true),
```

Comment:
since these new parameters aren't in the LaunchTemplateOptions defined above, you'll have to clear your LaunchTemplates that Karpenter generated to test. I think it's fine to leave it this way though. I'll be submitting a PR soon to adjust volume size and in that PR I'll implement a LaunchTemplate reaper so that templates get recycled more often.